pyaedt/modules/Mesh.py

```diff
@@ -390,14 +436,15 @@ def generate_mesh(self, name):
         Returns
         -------
         bool
-            ``True`` when successful, ``False`` when failed
+            ``True`` when successful, ``False`` when failed.
+            
         """
         return self.odesign.GenerateMesh(name) == 0
 
 
     @aedt_exception_handler
     def delete_mesh_operations(self, mesh_type=None):
-        """Remove mesh operations from a design. 
+        """Remove a mesh operations from a design. 
```

Comment:
```suggestion
        """Remove mesh operations from a design. 
```

Seems like it should be plural.